generator/src/main/java/com/scottlogic/deg/generator/App.java

```diff
@@ -1,14 +1,12 @@
 package com.scottlogic.deg.generator;
 
+import com.scottlogic.deg.generator.CommandLine.GenerateCommandLine;
 import picocli.CommandLine;
 
-import java.util.stream.Collectors;
-
 @CommandLine.Command(name = "dg")
-public class App implements Runnable {
+public class App {
```

Comment:
Should we leave this as implementing `Runnable` and keep the `Run` method for now?